docs/CONFIG-VARS.md

```diff
@@ -193,25 +193,26 @@ subnet_names = {
+| ssh_public_key                  |                                                                                     File name of public ssh key for jump and nfs VM | string |                                           "~/.ssh/id_rsa.pub" | Required with `create_jump_vm=true` or `storage_type=standard` |
+| cluster_api_mode                |                                                                                            Public or private IP for the cluster api | string |                                                      "public" | Valid Values: "public", "private" |
+| aks_cluster_private_dns_zone_id |                                                               Specifies private DNS zone resource ID for AKS private cluster to use | string |                                                            "" | For `cluster_api_mode=private` if `aks_cluster_private_dns_zone_id` is not specified then the value `System` is used else it is set to null. For details see [Configure a private DNS zone](https://learn.microsoft.com/en-us/azure/aks/private-clusters?tabs=azure-portal#configure-a-private-dns-zone) |
+| aks_cluster_sku_tier            |                         The SKU Tier that should be used for this Kubernetes Cluster. Optimizes api server for cost vs availability | string |                                                        "Free" | Valid Values:  "Free", "Standard" and "Premium" | 
+| cluster_support_tier            |                                                        Specifies the support plan which should be used for this Kubernetes Cluster. | string |                                          "KubernetesOfficial" |Possible values are `KubernetesOfficial` and `AKSLongTermSupport`. To enable long term K8s support is a combination of setting `aks_cluster_sku_tier` to `Premium` tier and explicitly selecting the `cluster_support_tier` as `AKSLongTermSupport`. For details see [Long term Support](https://learn.microsoft.com/en-us/azure/aks/long-term-support) and for which K8s version has long term support see [AKS Kubernetes release calendar](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar). | 
+| aks_cluster_run_command_enabled |                                                                                       Enable or disable the AKS Run Command feature |   bool |                                                         false | The AKS Run Command feature in AKS allows you to remotely execute commands within a running container of your AKS cluster directly from the Azure CLI or Azure portal. To enable the Run Command feature for an AKS cluster where Run Command is disabled, navigate to the Run Command tab for your AKS Cluster in the Azure Portal and select the Enable button. |
```

Comment:
Was there a need change to the existing part of the table, column widths and add the extra white space? Would prefer to keep the existing format the same as the other CONFIG-VARS.md tables if possible and just add the new entry. The new column format line made the entire table wider and harder to read all columns than before.